testapi.pm

```diff
@@ -971,11 +971,12 @@ sub assert_script_run {
             # not change default timeout.
             timeout      => 90,
             fail_message => '',
-            quiet        => testapi::get_var('_QUIET_SCRIPT_CALLS')
+            quiet        => testapi::get_var('_QUIET_SCRIPT_CALLS'),
+            type_speed => testapi::get_var('VNC_TYPING_LIMIT')
```

Comment:
This needs a default value:

```suggestion
            type_speed => testapi::get_var('VNC_TYPING_LIMIT', undef)
```

Reply:
Should this not be set always?

Reply:
The default is set here: https://github.com/os-autoinst/os-autoinst/blob/master/testapi.pm#L1408